src/components/TheTable.vue

```diff
@@ -1,16 +1,11 @@
 <template>
   <div v-show="TheItem" class="TheItemframe" @click="removeLTheItem()"></div>
   <TheItem
-    v-show="TheItem"
+    v-if="TheItem"
```

Comment:
@AlZorba I don't understand this one. It is testing that the component TheItem is defined ? I think I've read something like that in the doc. What is the intent ?

Reply:
@AlZorba ah ok ! For me this definitely need to be changed. The name should reflect what it contains, in camelCase, and also you have 2 variables with the same name, it's a miracle that it works :D